etl/steps/export/multidim/energy/latest/energy_prices.py

```diff
@@ -0,0 +1,49 @@
+from etl import multidim
```

Comment:
For me personally, showing monthly data as default would make much more sense. Yearly data don't show 2024 and drop in prices in 2023. It took me a while to discover there is monthly frequency.

Reply:
The problem with monthly data is that it only has one option: wholesale electricity prices. On the other hand, we have many more options for annual data. Using monthly as default would look like a very uninformative explorer, where most dropdowns have only one option.